src/EdFi.AnalyticsMiddleTier.Common/Component.cs

```diff
@@ -44,4 +44,4 @@ public enum Engine
         Postgresql,
         PostgreSQL
     }
-}
\ No newline at end of file
+}
```

Comment:
This file is has both issues in it, which were mentioned in the global comment: originally had a BOM at the beginning, and it needed an extra line break at the end of the file.